### ISSUE TITLE
Add a method for users to compute Radial profiles

### ISSUE THREAD

**Issue by garrettwrong** [ComputationalCryoEM/ASPIRE-Python#384]
In a few recent PRs (#383 ) we have had the need to generate simple radial profiles.

I found some examples of code to do this, but there are a variety of methods that yield different results. (Even picking a distance metric seems to have variety).

We'll need to decide what is the best method for ASPIRE, implement this method, and replace any ad hoc usage.

**Comment by janden** [comment]
I think the “canonical” solution here (given our use of FB bases) is to decompose of the radially symmetric components of an FB basis. There's a few things that are missing for this to work properly.

One is that we currently can only decompose over the entire basis, which is too expensive. We need to have the option of restricting the basis object further to only include a certain number of angular components (in this case one).

The other is that the basis is currently restricted to the unit disk in frequency and (sort of) the unit disk in space. If we're simply symmetrizing functions, we may want to extend this do beyond the unit disk. This poses another issue, but the user should have that as a default.

**Comment by garrettwrong** [comment]
Adding a note.  Users can generate line projections to accomplish the real-space alternative of this after #1150 .  Does not solve the original issue, but related and something to consider when addressing this issue.
